.gitignore

```diff
@@ -9,3 +9,5 @@
 dist/
 coverage/
 vendor/
+tags
+tags.*
```

Comment:
I don't think these are required on project specific `.gitignore`. These should be mentioned in your global git ignore https://docs.github.com/en/github/getting-started-with-github/getting-started-with-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer

Reply:
you are right! done it as such. thanks!